VULNERABILITY_RESPONSE_PROCESS.md

```diff
@@ -1,143 +0,0 @@
-# Monero Vulnerability Response Process
-
-## Preamble
-
-Researchers/Hackers: while you research/hack, we ask that you please refrain from committing the following:
-- Denial of Service / Active exploiting against the network
-- Social Engineering of Monero staff or contractors
-- Any physical or electronic attempts against Monero community property and/or data centers
-
-## I. Point of Contacts for Security Issues
-
-```
-ric@getmonero.org
-BDA6 BD70 42B7 21C4 67A9  759D 7455 C5E3 C0CD CEB9
-
-luigi1111@getmonero.org
-8777 AB8F 778E E894 87A2  F8E7 F4AC A018 3641 E010
-
-moneromooo.monero@gmail.com
-48B0 8161 FBDA DFE3 93AD  FC3E 686F 0745 4D6C EFC3
```

Comment:
Hmm, I hadn't actually noticed that this was here. I never check mail...

Reply:
Is there another address you'd like to use instead?

Reply:
No.